sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_base_client.py

```diff
@@ -136,11 +137,7 @@ def send_request(self, request: HttpRequest, *, stream: bool = False, **kwargs)
         api_version = self._api_version
         if hasattr(api_version, "value"):
             api_version = api_version.value
-        if self._api_version.startswith("v"):
-            # Skip the API version formatting for v2 API versions
-            request_copy = deepcopy(request)
-        else:
-            request_copy = _format_api_version(request, api_version)
+        request_copy = _format_api_version(request, api_version)
```

Comment:
If we call `_format_api_version` with a client configured for v2.0/v2.1, won't that add an API version query parameter? Even if the service ignores the parameter and uses the version from the path, I'm not sure we should be sending that.

Reply:
It will add an API version query to url in this case. It would be useful when a client configured for v2.0 want to make a call with v2.1 API version, so I think we should send that for v2 APIs.

Reply:
Has proved that the api_version in query parameter is ignored in service, so it's not necessary to format the api version for clients with v2 versions.